### PR TITLE
Cors filter add

### PR DESCRIPTION
**Pull request by arifratul41**

### CODE REVIEW

File: src/main/java/com/ideal/studentlog/config/CORSFilter.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
```
Comment (ishmum123) [inline src/main/java/com/ideal/studentlog/config/CORSFilter.java]: Can we use `@Configuration` annotation?

File: src/main/java/com/ideal/studentlog/config/CORSFilter.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Order(Ordered.HIGHEST_PRECEDENCE)
```
Comment (ishmum123) [inline src/main/java/com/ideal/studentlog/config/CORSFilter.java]: What happens if we remove this line?

**Review by ishmum123 — CHANGES_REQUESTED** [review]

File: src/main/java/com/ideal/studentlog/config/CORSFilter.java (hunk 1 above)
Comment (arifratul41) [reply src/main/java/com/ideal/studentlog/config/CORSFilter.java]: Can access the APIs from next.js without any issue after changing the annotation to @Configuration

File: src/main/java/com/ideal/studentlog/config/CORSFilter.java (hunk 2 above)
Comment (arifratul41) [reply src/main/java/com/ideal/studentlog/config/CORSFilter.java]: Can access the APIs from next.js without any issue even after removing the line

**Pull request merged**